.drone.yml

```diff
@@ -8,7 +8,21 @@ pipeline:
       - apk add --no-cache bash sed grep bc coreutils curl
       - ./ci-build.sh
     when:
-      event: [push, pull_request]
+      event: push
+
+  scan-clamav:
+    image: quay.io/ukhomeofficedigital/anchore-submission:latest
+    image_name: clamav
+    local_image: true
+    when:
+      event: push
```

Comment:
Do we not want this on tag too?
Example scenario: Nothing needs to be changed in the repo but a new tag is made so that the `yum update` step in the Dockerfile runs again and dependancies are updated.

I guess we trust that if someone needs to do that then they'll re-run the build that was pushed to master which would scan the image again...

Reply:
Good point, but at the point of tagging the intentions are quite clear that the user will want an image to be created and pushed regardless of the result (may be off master or different commit hashes), which is why I left it off.

Reply:
Fair enough 👍